### PR TITLE
Rename `application-cli` page, update command reference & examples

### LINKED ISSUE #1700: Rename page `application-cli.adoc` to use new terminology
### Description
- Rename page `application-cli.adoc` to `client-cli.adoc` in the SDK docs.
- Update all the links to the page in the documentation
- Update example `generator status` on page: it should only return one validator, not all 103
- Remove outdated account topic, and double check for other outdated content on the page.
- Add Passphrase:create and check for any changes in the CLI
- Verify validity of commands and examples according to #1626 and #1622 

#### Affected branch / URL
`docs-sdk`, `development`

#### Page location
`application-cli.adoc`

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1700

### CODE REVIEW

File: modules/ROOT/pages/client-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,7518 @@
...
+----
+
+.Example
+[%collapsible]
+====
+[source,bash]
+----
+$./bin/run auto-complete
+ ›   Error: command auto-complete not found
+$./bin/run autocomplete bash
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/client-cli.adoc]:
```suggestion
$ ./bin/run autocomplete bash
```

File: modules/ROOT/pages/client-cli.adoc [hunk 2]
```diff
@@ -0,0 +1,7518 @@
...
+FLAGS
+  -d, --data-path=<value>  Directory path to specify where node data is stored. Environment variable "LISK_DATA_PATH" can also be used.
+  --pretty                 Prints JSON in pretty format rather than condensed.
+
+DESCRIPTION
+  Get block information for a given id or height.
+
+EXAMPLES
+  block:get e082e79d01016632c451c9df9276e486cb7f460dc793ff5b10d8f71eecec28b4
+  block:get 2
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/client-cli.adoc:270]:
```suggestion
  ./bin/run block:get e082e79d01016632c451c9df9276e486cb7f460dc793ff5b10d8f71eecec28b4
  ./bin/run block:get 2
```

File: modules/ROOT/pages/client-cli.adoc [hunk 3]
```diff
@@ -0,0 +1,7518 @@
...
+  ./bin/run block:get  Get block information for a given id or height.
+----
+
+==== block:get
+Gets block information for a given id or height.
+
+[source,bash]
+----
+USAGE
+  $ ./bin/run block get [INPUT] [-d <value>] [--pretty]
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/client-cli.adoc]:
```suggestion
  $ ./bin/run block:get [INPUT] [-d <value>] [--pretty]
```

File: modules/ROOT/pages/client-cli.adoc [hunk 4]
```diff
@@ -0,0 +1,7518 @@
...
+                        "dataType":"string",
+                        "format":"lisk32"
+                     }
+                  },
+                  "required":[
+                     "address"
+                  ]
+               },
+               "response":{
+
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/client-cli.adoc:7408]:
```suggestion

DESCRIPTION
  Get transaction from local node by ID.

```

Comment (TalhaMaliktz) [inline modules/ROOT/pages/client-cli.adoc]:
```suggestion
  $ ./bin/run transaction:get [ID] [-d <value>] [--pretty]
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Looks good, just some suggestions. 
Please also create the issue for updating CLI again with reference to `passphrase:create` command.

File: modules/ROOT/pages/client-cli.adoc [hunk 5]
```diff
@@ -0,0 +1,7518 @@
...
+  endpoint       Commands relating to sidechain endpoint.
+  generator      Commands relating to sidechain block generator.
+  genesis-block  Creates genesis block file.
+  keys           Commands relating to sidechain key generation.
+  node           Get node information from a running application.
+  passphrase     Commands relating to sidechain passphrases.
+  transaction    Commands relating to sidechain transactions.
+
+COMMANDS
+  autocomplete  display autocomplete installation instructions
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:48]:
```suggestion
  autocomplete  Display autocomplete installation instructions.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 6]
```diff
@@ -0,0 +1,7518 @@
...
+=== hash-onion
+Creates hash onions to be used by a generator.
+
+[source,bash]
+----
+USAGE
+  $ ./bin/run hash-onion [-o <value>] [-c <value>] [-d <value>] [--pretty]
+
+FLAGS
+  -c, --count=<value>     [default: 1000000] Total number of hashes to produce
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:144]:
```suggestion
  -c, --count=<value>     [default: 1000000] Total number of hashes to produce.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 7]
```diff
@@ -0,0 +1,7518 @@
...
+  -d, --distance=<value>  [default: 1000] Distance between each hashes
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:145]:
```suggestion
  -d, --distance=<value>  [default: 1000] Distance between each hash.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 8]
```diff
@@ -0,0 +1,7518 @@
...
+  -o, --output=<value>    Output file path
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:146]:
```suggestion
  -o, --output=<value>    Output file path.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 9]
```diff
@@ -0,0 +1,7518 @@
...
+  --enable-faucet-plugin              Enable Faucet Plugin. Environment variable "LISK_ENABLE_FAUCET_PLUGIN" can also be used.
+  --enable-forger-plugin              Enable Forger Plugin. Environment variable "LISK_ENABLE_FORGER_PLUGIN" can also be used.
+  --enable-monitor-plugin             Enable Monitor Plugin. Environment variable "LISK_ENABLE_MONITOR_PLUGIN" can also be used.
+  --enable-report-misbehavior-plugin  Enable ReportMisbehavior Plugin. Environment variable "LISK_ENABLE_REPORT_MISBEHAVIOR_PLUGIN" can also be used.
+  --faucet-plugin-port=<value>        Port to be used for Faucet Plugin. Environment variable "LISK_FAUCET_PLUGIN_PORT" can also be used.
+  --monitor-plugin-port=<value>       Port to be used for Monitor Plugin. Environment variable "LISK_MONITOR_PLUGIN_PORT" can also be used.
+  --monitor-plugin-whitelist=<value>  List of IPs in comma separated value to allow the connection. Environment variable "LISK_MONITOR_PLUGIN_WHITELIST" can also be used.
+  --overwrite-config                  Overwrite network configs if they exist already
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:209]:
```suggestion
  --overwrite-config                  Overwrite network configs if they exist already.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 10]
```diff
@@ -0,0 +1,7518 @@
...
+----
+USAGE
+  $ ./bin/run blockchain:import [FILEPATH] [-d <value>] [-f]
+
+ARGUMENTS
+  FILEPATH  Path to the gzipped blockchain data.
+
+FLAGS
+  -d, --data-path=<value>  Directory path to specify where node data is stored. Environment variable "LISK_DATA_PATH" can also be used.
+  -f, --force              Delete and overwrite existing blockchain data
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:404]:
```suggestion
  -f, --force              Delete and overwrite existing blockchain data.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 11]
```diff
@@ -0,0 +1,7518 @@
...
+==== config:create
+Creates a configuration file for the blockchain application.
+
+[source,bash]
+----
+USAGE
+  $ ./bin/run config:create -i <value> [-o <value>] [-l <value>]
+
+FLAGS
+  -i, --chain-id=<value>  (required) ChainID in hex format. For example, Lisk mainnet mainchain is 00000000
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:482]:
```suggestion
  -i, --chain-id=<value>  (required) ChainID in hex format. For example, Lisk mainnet mainchain is 00000000.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 12]
```diff
@@ -0,0 +1,7518 @@
...
+  -l, --label=<value>     App Label
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:483]:
```suggestion
  -l, --label=<value>     App label.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 13]
```diff
@@ -0,0 +1,7518 @@
...
+  -o, --output=<value>    Directory where the config file is saved
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:484]:
```suggestion
  -o, --output=<value>    Directory where the config file is saved.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 14]
```diff
@@ -0,0 +1,7518 @@
...
+==== endpoint:invoke
+Invokes the provided endpoint.
+
+[source,bash]
+----
+USAGE
+  $ ./bin/run endpoint:invoke [ENDPOINT] [PARAMS] [-d <value>] [--pretty] [-f <value>]
+
+ARGUMENTS
+  ENDPOINT  Endpoint to invoke
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:606]:
```suggestion
  ENDPOINT  Endpoint to invoke.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 15]
```diff
@@ -0,0 +1,7518 @@
...
+  PARAMS    Endpoint parameters (Optional)
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:607]:
```suggestion
  PARAMS    Endpoint parameters (Optional).
```

File: modules/ROOT/pages/client-cli.adoc [hunk 16]
```diff
@@ -0,0 +1,7518 @@
...
+FLAGS
+  -d, --data-path=<value>         Directory path to specify where node data is stored. Environment variable "LISK_DATA_PATH" can also be used.
+  -w, --password=<value>          Specifies a source for your secret password. Command will prompt you for input if this option is not set.
+                                  	Examples:
+                                  	- --password=pass:password123 (should only be used where security is not important)
+  --height=<value>                Last generated block height.
+  --max-height-generated=<value>  Validator's largest previously generated height.
+  --max-height-prevoted=<value>   Validator's largest prevoted height for a block.
+  --pretty                        Prints JSON in pretty format rather than condensed.
+  --use-status-value              Use status value from the connected node
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:740]:
```suggestion
  --use-status-value              Use status value from the connected node.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 17]
```diff
@@ -0,0 +1,7518 @@
...
+Imports generator's data from a file.
+
+[source,bash]
+----
+USAGE
+  $ ./bin/run generator:import -f <value> [-d <value>] [--pretty]
+
+FLAGS
+  -d, --data-path=<value>  Directory path to specify where node data is stored. Environment variable "LISK_DATA_PATH" can also be used.
+  -f, --file-path=<value>  (required) Path of the file to import from
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:831]:
```suggestion
  -f, --file-path=<value>  (required) Path of the file to import from.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 18]
```diff
@@ -0,0 +1,7518 @@
...
+Creates a genesis block file and the corresponding generator\'s configuration data and accounts information.
+
+[source,bash]
+----
+USAGE
+  $ ./bin/run genesis-block:create -f <value> [-n <value>] [-c <value>] [-o <value>]
+
+FLAGS
+  -c, --config=<value>       File path to a custom config. Environment variable "LISK_CONFIG_FILE" can also be used.
+  -f, --assets-file=<value>  (required) Path to file which contains genesis block asset in JSON format
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:914]:
```suggestion
  -f, --assets-file=<value>  (required) Path to file which contains genesis block asset in JSON format.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 19]
```diff
@@ -0,0 +1,7518 @@
...
+  -n, --network=<value>      [default: default] Default network config to use. Environment variable "LISK_NETWORK" can also be used.
+  -o, --output=<value>       [default: config] Output folder path of the generated genesis block
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:916]:
```suggestion
  -o, --output=<value>       [default: config] Output folder path of the generated genesis block.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 20]
```diff
@@ -0,0 +1,7518 @@
...
+==== keys:create
+Return keys corresponding to the given passphrase.
+
+[source,bash]
+----
+USAGE
+  $ ./bin/run keys:create [-o <value>] [-p <value>] [-n] [-w <value>] [-c <value>] [-f <value>] [-i <value>]
+
+FLAGS
+  -c, --count=<value>       [default: 1] Number of keys to create
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:978]:
```suggestion
  -c, --count=<value>       [default: 1] Number of keys to create.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 21]
```diff
@@ -0,0 +1,7518 @@
...
+  -f, --offset=<value>      Offset for the key derivation path
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:979]:
```suggestion
  -f, --offset=<value>      Offset for the key derivation path.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 22]
```diff
@@ -0,0 +1,7518 @@
...
+  -i, --chainid=<value>     Chain id
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:980]:
```suggestion
  -i, --chainid=<value>     Chain id.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 23]
```diff
@@ -0,0 +1,7518 @@
...
+  -n, --no-encrypt          No encrypted message object to be created
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:981]:
```suggestion
  -n, --no-encrypt          No encrypted message object to be created.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 24]
```diff
@@ -0,0 +1,7518 @@
...
+==== keys:encrypt
+Return keys corresponding to the given passphrase.
+
+[source,bash]
+----
+USAGE
+  $ ./bin/run keys:encrypt -f <value> [-w <value>]
+
+FLAGS
+  -f, --file-path=<value>  (required) Path of the file to encrypt from
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:1069]:
```suggestion
  -f, --file-path=<value>  (required) Path of the file to encrypt from.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 25]
```diff
@@ -0,0 +1,7518 @@
...
+  -w, --password=<value>   Specifies a source for your secret password. Command will prompt you for input if this option is not set.
+                           	Examples:
+                           	- --password=pass:password123 (should only be used where security is not important)
+
+DESCRIPTION
+  Encrypt keys from a file and overwrite the file
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:1075]:
```suggestion
  Encrypt keys from a file and overwrite the file.
```

File: modules/ROOT/pages/client-cli.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/client-cli.adoc:270]: as this is copied from the CLI, I would leave the example as is

File: modules/ROOT/pages/client-cli.adoc [hunk 26]
```diff
@@ -0,0 +1,7518 @@
...
+Import keys from a file.
+
+[source,bash]
+----
+USAGE
+  $ ./bin/run keys:import -f <value> [-d <value>] [--pretty]
+
+FLAGS
+  -d, --data-path=<value>  Directory path to specify where node data is stored. Environment variable "LISK_DATA_PATH" can also be used.
+  -f, --file-path=<value>  (required) Path of the file to import from
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:1169]:
```suggestion
  -f, --file-path=<value>  (required) Path of the file to import from.
```

File: modules/ROOT/pages/client-cli.adoc [hunk 27]
```diff
@@ -0,0 +1,7519 @@
...
+                        "format":"lisk32"
+                     }
+                  },
+                  "required":[
+                     "address"
+                  ]
+               },
+               "response":{
+                  "$id":"/validators/endpoint/getValidatorResponse",
+
```
Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:7211]:
```suggestion
  ./bin/run passphrase:encrypt  Encrypt the secret passphrase using the password.
```

Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:7268]:
```suggestion
  Encrypt the secret passphrase using the password.
```

Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:7343]:
```suggestion
  -a, --params=<value>               Creates transaction with specific params information.
```

Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:7349]:
```suggestion
  -k, --key-derivation-path=<value>  [default: m/44'/134'/0'] Key derivation path to use to derive keypair from passphrase.
```

Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:7353]:
```suggestion
  -s, --sender-public-key=<value>    Set a custom senderPublicKey property for the transaction, to be used when account address does not correspond to signer's private key.
```

Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:7355]:
```suggestion
  --no-signature                     Creates the transaction without a signature. Your passphrase will therefore not be required.
```

Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:7359]:
```suggestion
  --send                             Create and immediately send transaction to a node.
```

Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:7451]:
```suggestion
  TRANSACTION  A transaction to be sent to the node encoded as hex string.
```

Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:7483]:
```suggestion
  TRANSACTION  The transaction to be signed encoded as hex string.
```

Comment (chris529) [inline modules/ROOT/pages/client-cli.adoc:7488]:
```suggestion
  -k, --key-derivation-path=<value>  [default: m/44'/134'/0'] Key derivation path to use to derive keypair from passphrase.
```

**Review by chris529 — COMMENTED** [review]
Looks great! Just a few very minor points 👍

**Comment by Tschakki** [comment]
Hey @chris529 , I see your comments are all related to content which was copied from the client CLI. That is why I won't update it here in this PR. If you feel this should be updated, please create a corresponding issue in the lisk-sdk repo, thx.

**Comment by chris529** [comment]
@Tschakki Hey Mona, Yes I think most of the comments were related to the client CLI. Agreed and understood, so I will approve it.👍

**Review by shuse2 — APPROVED** [review]
Generally looks good. As @TalhaMaliktz mentioned, `node` command should be `system` (it will also need to be fixed in SDK as well

**Review by TalhaMaliktz — APPROVED** [review]
LGTM 🎉

**Pull request merged**